### DISCUSSION TITLE
NPM Publish

### DISCUSSION THREAD

**Discussion by dannyd30** [aws/aws-msk-iam-sasl-signer-js#22] [General]
Hi, is there any plans to publish this package to the NPM registry in near future? We are in need of this package for one of our services in production, but couldn't find this on NPM registry. Would highly appreciate if we can publish package in the registry.

**Comment by sankalpbhatia** [comment]
Hey, thanks for raising this. 

We do have this published in npm: https://www.npmjs.com/package/aws-msk-iam-sasl-signer-js. Does this not work for you?

**Reply by dannyd30** [reply]
Thanks for the response, I can install it now. For some reason, it doesn't show up in google. Sorry for the confusion.

**Reply by sankalpbhatia** [reply]
No worries. Happy to help.
